beesdoo_product_pricelist/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 - BEES coop SCRLfs
+#   - Vincent Van Rossem <vincent@coopiteasy.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "beesdoo_product_pricelist",
+    "summary": """
+        Allow quick product price editing
+        on point of sale, purchase and sale modules""",
+    "author": "Coop IT Easy SCRLfs",
+    "website": "https://github.com/beescoop/Obeesdoo",
+    "category": "Sales",
+    "version": "12.0.1.0.0",
+    "depends": ["beesdoo_product", "point_of_sale", "purchase", "sale"],
```

Comment:
@vvrossem do the dependency on beesdoo product is necessary. if we go with the suggestion of @robinkeunen I would rather see having beesdoo_product depending on this one (As far as it's possible).

Reply:
It's necessary because `beesdoo_product` declares the field `main_seller_id`